src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -556,6 +618,14 @@ def update_container_settings(cmd, resource_group_name, name, docker_registry_se
         update_app_settings(cmd, resource_group_name, name, settings, slot)
     settings = get_app_settings(cmd, resource_group_name, name, slot)
 
+    if bool(multicontainer_config_file) and bool(multicontainer_config_type):
+        encoded_config_file = _get_linux_multicontainer_config_file(cmd, resource_group_name,
+                                                                    name, multicontainer_config_file)
+        linux_fx_version = _format_linux_fx_version(encoded_config_file, multicontainer_config_type)
+        update_site_configs(cmd, resource_group_name, name, linux_fx_version=linux_fx_version)
+    else:
+        raise CLIError('Must use --multicontainer-config-file FILE --multicontainer-config-type TYPE')
```

Comment:
This could be very wrong, unless I mis-understood. To me now this command would only succeed with `multicontainer_config_file` and `multicontainer_config_type`; while all other settings would not go through